src/include/kdbprivate.h

```diff
@@ -24,6 +24,32 @@
 
 #include <limits.h>
 
+#define KDB_STRING_TO_BOOLEAN(string) (kdb_boolean_t) !strcmp (string, "1")
```

Comment:
Maybe better in kdbmacros.h? Is this useful without any error handling? Prefer to prefix with ELEKTRA_